### DISCUSSION TITLE
2.13.1 breaks "select inside <character>" with leading `#` on line

### DISCUSSION THREAD

**Discussion by mbspng** [JetBrains/ideavim#933] [General]
2.13.1 breaks "select inside <character>" with leading `#` on line. e.g. `vi(` . Same for `ci[` and so on.

**Comment by citizenmatt** [comment]
Could you provide an example, please?

**Reply by mbspng** [reply]
```
# bla(bla bla)
```

**Reply by citizenmatt** [reply]
Thanks. This appears to be a bug with select inside when used in a comment block. See [VIM-3538](https://youtrack.jetbrains.com/issue/VIM-3538)
